### PR TITLE
Remove error message when editing/creating slugs

### PR DESCRIPTION
**Pull request by cohencodes**

### CODE REVIEW

File: src/plugins/ncr/components/create-node-modal/AbstractDelegate.js [hunk 1]
```diff
@@ -159,24 +158,6 @@ export default class AbstractDelegate {
...
 
-  /**
-   * @link https://redux-form.com/7.4.2/docs/api/reduxform.md
-   *
-   * @param {Object} values
-   * @param {Object} formProps
-   *
-   * @returns {Object}
-   */
-  handleValidate(values, formProps) {
```
Comment (efjacobson) [inline src/plugins/ncr/components/create-node-modal/AbstractDelegate.js]: @cohencodes there are other things that need to be validated. title for ex

File: src/plugins/ncr/components/slug-editor/index.jsx [hunk 2]
```diff
@@ -123,13 +124,18 @@ class SlugEditor extends React.Component {
...
       };
-    }, this.validateSlug);
+    });
   }
 
   handleRename() {
     const { delegate, initialSlug } = this.props;
     const { slug } = this.state;
 
+    if (slug === initialSlug) {
```
Comment (efjacobson) [inline src/plugins/ncr/components/slug-editor/index.jsx]: @cohencodes why not just disable the button?

**Comment by cohencodes** [comment]
@efjacobson ready for review

File: src/plugins/ncr/services/SluggableSubscriber.js [hunk 3]
```diff
@@ -87,35 +86,7 @@ export default class SluggableSubscriber extends EventSubscriber {
    * @param {FormEvent} formEvent
    */
   onValidateForm(formEvent) {
```
Comment (efjacobson) [inline src/plugins/ncr/services/SluggableSubscriber.js]: @cohencodes do we still need this?

Comment (cohencodes) [reply src/plugins/ncr/services/SluggableSubscriber.js]: @efjacobson no I took this out

**Comment by efjacobson** [comment]
looks good to me, @gdbrown can you see anything here that might bite us?

**Pull request merged**